### PR TITLE
Reduce amount of pain from scalding reagents

### LINKED ISSUE #34975: cup ramen now knocks you out due to its temperature
(WEB REPORT BY: locese REMOTE: 206.221.180.138:7777)
# Revision
f50f58e093be7d81c113c9e40c8908bce89088bf

# Description
see title
# Steps to Reproduce
buy a cup ramen from the vendomat
pour water so it heats
drink it
# What you Expected
a tasty cup of ramen
# What Actually Happened
paincrit, burnt head

#34842 is at fault for this

### PR DESCRIPTION
**Pull request by Hinaichigo**
## What this does
This makes it so far less pain is applied when you get scalded with hot reagents. Currently, it does 5 times the burn damage, as temporary pain-damage. This reduces it to 1x, so greatly lessens the pain effect of being splashed with hot oil or scarfing down a large amount of hot food.

## Why it's good
Make eating scalding hot ramen or caramel less of a problem in terms of temporarily passing out from pain.

This is one option which can be considered an alternative to #35191

Fixes  #34975

## Changelog
:cl:
 * tweak: Reduced pain damage from being splashed by or consuming scalding hot reagents.

### CODE REVIEW

**Comment by wotisjanitor** [comment]
PR does not revert previous temperature changes

**Comment by Hinaichigo** [comment]
@wotisjanitor 
https://github.com/vgstation-coders/vgstation13/pull/35189#issuecomment-1765346597

**Comment by ValkyrieSkies** [comment]
The alternative PR is pretty dumb and inconsistent so this is a way more sensible option.

**Comment by githubuser4141** [comment]
Aw, couldn't it be lowered to x2 or x3 instead? Aren't large superficial burns supposed to be painful?

Wait, why not give the ramen cups a small satchet of water or dried water (its the future) that you're supposed to add after it's cooked, to cool it by diluting the hot water with cold water? This'd retain the feature of being punished for eating in a too hasty way without forcing people to actually wait minutes for it to cool. Or, perhaps add sinks on every map for people to dunk their piping hot ramen under? This could eventually be expanded into a proper water plumbing system.

**Comment by Hinaichigo** [comment]
Personally I like it as is as well, but I'm trying to root out the general source of the dislike of the reagent heat related stuff so that the drastic step of reverting the (unrelated) cooling system can potentially be avoided in #35189. As somehow there has developed a connection or association in people's minds between the two. It's something like making a ritual sacrifice to assuage the vibes is the best I can understand it as. In the most sincere way this can be said, sometimes rationality falls by the wayside and people are swept into movements and compulsions based on these sorts of things. Like how they would sacrifice animals before battle to read the entrails in the olden days, type of thing.

For the calcium thing, if you think about it having the calcium oxide not completely consume as much water might make sense. Typically if you dissolve it in water it's not going to be a saturated sludge of calcium hydroxide it's going to be a basic solution of bitter water. Or yea having a little water activated heat pack built into the cup could make sense too.

**Comment by Hinaichigo** [comment]
Also, you already can add more water than necessary to the ramen after it cooks to cool it down a bit.

**Comment by kacurasuo** [comment]
is this pr made correctly? the code for handling thermal dissipation is missing from this pr?
I checked out the pr to test it but the chemistry-holder.dm has the reversion of #34855 from #35189 included?
This pr's version of chemistry-holder.dm is 1103 lines and the current live version is 1233 but the diff only shows 1 line changed?
Maybe I just don't get everything that git's doing here but I'm very confused and it makes testing this PR very challenging because the environment <-> reagent heat transfer just isn't working at all

**Comment by Hinaichigo** [comment]
The heat transfer between reagents and air (added in #34855), and the scalding mechanics (added in #34842) are two different things. Right now there is confusion over them both partly because of some reverted PRs from a while ago that are not currently relevant being conflated with the current situations in people's memories, and because of #35189 also targeting the wrong PR for revert but somehow snowballing support due to various things getting crossed in people's minds and me trolling the thread a little bit about inflatable shelters, wotisjanitor rallying people on the other side, etc. As I mentioned, rationality has fallen by the wayside to an extent in this case, and instead given way to spectacle.

So basically #35189 would only make reagents stay at their current temperatures and not ambiently adjust temperatures, and re-introduce a small quirk/bug. It would have no direct impact on the scalding damage as it mentions in its PR body.

**Comment by kacurasuo** [comment]
no, I mean, when I checked out this PR, the code handling thermal dissipation was missing
if you explore the file that bit of code is missing but it's not flagged in the diff?
maybe git is trying to be helpful and base this pr around the currently opened reversion?
image

**Comment by Hinaichigo** [comment]
Make sure you're looking at the correct branch/commit maybe?

**Comment by kacurasuo** [comment]
ok, I figured it out, the branch that you put this commit on was out of date, so it doesn't have #34855.
that's not a problem for this pr, except it does make it harder to test

**Comment by SonixApache** [comment]
> Also, you already can add more water than necessary to the ramen after it cooks to cool it down a bit.

yeah that's real fucking funny, go ahead and try doing that

**Comment by smashedandslammed** [comment]
Just revert the whole scalding mechanic, heated smoke is the only thing lethal at this moment and you may as well just use polysmoke because a stasis beaker and laser scalpel is needed to make superhot caramel smoke.
